app/src/main/kotlin/dev/patrickgold/florisboard/lib/devtools/Devtools.kt

```diff
@@ -34,6 +36,16 @@ import java.io.InputStreamReader
 @Suppress("MemberVisibilityCanBePrivate")
 object Devtools {
     fun generateDebugLog(context: Context, prefs: AppPrefs? = null, includeLogcat: Boolean = false): String {
+        return buildString {
+            generateDebugLogHeader(context, prefs)
```

Comment:
```suggestion
            append(generateDebugLogHeader(context, prefs))
```
I just messed up in the refactoring process.